Rakefile

```diff
@@ -108,9 +108,19 @@ Dir[File.expand_path("../*gemspec", __FILE__)].reverse.each do |gemspec_path|
   Gem::PackageTask.new(gemspec).define
 end
 
+def with_clean_env(&block)
+  if defined?(Bundler)
+    Bundler.with_clean_env(&block)
```

Comment:
Just trying to think of weird issues - is there anytime Bundler wouldn't have `with_clean_env` defined?  It was added back in 2010 so I'm guessing not...